content/english/Kusto-KC7/_index.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "KC7: Cyber Defender Game"
+description: "KC7 is a game that teaches you real world cybersecurity skills used by professional Cyber Defenders"
+date: 2023-02-1T13:24:17-07:00
+prereq: "None"
+difficulty: "Advanced"
+draft: false
+icon: "fas fa-database"
+weight: 131
+---
+
+# Introduction: Learn Kusto Query Language (KQL) to Catch the Hacker!
+
+![logo](./Images/KC7Logo.png)
+
+## Welcome new Cyber Defenders! 
+*---Threat intel is for everyone*
+Learn Cybersecurity using real threat data.
+**KC7 is a game that teaches you real world cybersecurity skills**
+>KC7 simulates an intrusion by multiple cyber threat actors against a fictitious company that teaches you to identify adversary activity across all seven phases of the Cyber Kill Chain using realistic data.
+<!--![killchain](../Kusto-KC7/Images/KillChain.jpg)-->
+<!--![alt](https://www.oceanpointins.com/wp-content/uploads/2020/02/canstockphoto26807912.jpg)-->
+
+By the end of your first day on the job, you should be able to: 
+ 
+{{<icon name="ok" size="large">}} Use Azure Data Explorer (web version)   
+{{<icon name="ok" size="large">}} Use multiple data sets to answer targeted questions  
+{{<icon name="ok" size="large">}} Find cyber activity in logs including: email, web traffic, and server logs  
+{{<icon name="ok" size="large">}} Use multiple techniques to track the activity of APTs (Advanced Persistent Threats)  
+{{<icon name="ok" size="large">}} Use third party data sets to discover things about your attackers   
+{{<icon name="ok" size="large">}} Build a threat intelligence report    
+{{<icon name="ok" size="large">}} Make recommendations on what actions a company can take to protect themselves    
+
+If you are a trainer please start with the next block **T3: Train-the-Trainer Intro**. If you are a student please skip ahead to **Activity 0: Login & Setup of the Cyber Environment**
```

Comment:
As discussed let's move the trainer pages to the end of the activity and we can simply omit this paragraph

Reply:
I think it's still necessary, but i reworded it to fit with T3 being moved to the bottom